src/components/RouteOption/DockedEbikeRouteOption.jsx

```diff
@@ -16,8 +17,8 @@ export default function DockedEbikeRouteOption({
   const toDockingStationId = routeOption.selectedStationIds.destination;
 
   function handleArticleClick(event) {
+    // Do not send click when clicking on an input element
```

Comment:
oops - still not in its original place. changing this will link this change to this pull request; might might be confusing if we scroll through git blame / history. my bad, it was _below_ the if statement, not above. should be 

```javascript
if (event.target instanceof HTMLInputElement) {
  // Do not send click when clicking on an input element
```